lib/LDAPUserManager.php

```diff
@@ -265,6 +265,14 @@ public function ensureAttribute(array &$ldif, string $attribute, string $fallbac
 	}
 
 	public function buildNewEntry($username, $password, $base) {
+		// Make sure the parameters don't fool the following algorithm
+		if (strpos($username, PHP_EOL) !== false)
```

Comment:
please use { curly brackets } even if it is just one (according to our coding styles)